dev/benchmarks/macrobenchmarks/README.md

```diff
@@ -4,6 +4,33 @@ Performance benchmarks use either flutter drive or the web benchmark harness.
 
 ## Mobile benchmarks
 
+### Cull opacity benchmark
```

Comment:
Your PR here is adding this text which I don't see on master, maybe this got deleted in a recent change and when merging back in master you accidentally restored it?

Reply:
I guess but what I didi isn’t fetch all new things and see what was changed so I changed nothing but I will make sure

Reply:
When there's a merge conflict you have to manually look through all the changes and decide which makes sense to keep. [This change](https://github.com/flutter/flutter/commit/7bef51acc979ee467c02a7503a8990a6edbb0ae3) removed the this README text, so with your merge conflict you need to also remove it.

Reply:
@christopherfujino You merged it like this, right?